src/js/main.js

```diff
@@ -4,6 +4,11 @@ import Icons from '../elements/icons/Icons';
 import Feed from '../objects/feed/Feed';
 import Toggle from '../utilities/toggle/Toggle';
 import Track from '../utilities/track/Track';
+
+import ChartBar from 'nyco-patterns/src/objects/charts/chart--bar';
+import ChartHorizontalBar from 'nyco-patterns/src/objects/charts/chart--horizontal-bar';
+import ChartLine from 'nyco-patterns/src/objects/charts/chart--line';
+import ChartPie from 'nyco-patterns/src/objects/charts/chart--pie';
```

Comment:
I haven't adopted the double dash for .js files to define different types (consistently!). I haven't seen many conventions in .js where this would be expected so I would normally stick to just single dashes for filenames. Double dashes work pretty well when reading css classes in markup and you need to distinguish between different types of a certain style (as well as double underscore for nested children). Open to stylelistic suggestions.

Reply:
No preference, I saw that `--` was used in another file so I ran with that. Personally I'd prefer using camal case or `-`.

Reply:
Can the paths be updated to '../objects'? There are reference errors I'm getting.

Reply:
yep! will update.